test/system-contracts/hedera-account-service/ihrc-632/aliasAccountUtility.js

```diff
@@ -94,7 +94,7 @@ describe('@HAS IHRC-632 Test Suite', () => {
       ); // evm address
     });
 
-    it('Should execute getEvmAddressAliasPublic with not long zero address and get INVALID_ACOUNT_ID', async () => {
```

Comment:
This test had been verifying incorrect behavior which has been fixed here - https://github.com/hiero-ledger/hiero-consensus-node/issues/17570 and was therefore failing after updating the image tag